mdoc/src/main/scala/mdoc/internal/cli/Settings.scala

```diff
@@ -66,7 +66,9 @@ case class Settings(
     @Section("Compiler options")
     @Description(
       "Classpath to use when compiling Scala code examples. " +
-        "Defaults to the current thread's classpath."
+        "Defaults to the current thread's classpath." +
+        "You can use Coursier's fetch command to generate the classpath for you:" +
+        "`--classpath $(cs fetch org::artifact:version | tr '\n' ':')`"
```

Comment:
Coursier has a “-p” flag to insert the colon on Unix systems and a semicolon on window.

Reply:
Thanks for letting me know, I was not aware. I tried it and it worked fine but at one point, I had extra information printed to the console as well. Consider the following:

```
cs fetch -p co.fs2::fs2-io:2.2.2
https://repo1.maven.org/maven2/org/scala-lang/scala-library/maven-metadata.xml
  No new update since 2019-09-18 09:57:24
/Users/david/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/co/fs2/fs2-io_2.13/2.2.2/fs2-io_2.13-2.2.2.jar:/Users/david/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/scala-lang/scala-library/2.13.1/scala-library-2.13.1.jar:/Users/david/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/co/fs2/fs2-core_2.13/2.2.2/fs2-core_2.13-2.2.2.jar:/Users/david/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/typelevel/cats-core_2.13/2.1.0/cats-core_2.13-2.1.0.jar:/Users/david/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/typelevel/cats-effect_2.13/2.0.0/cats-effect_2.13-2.0.0.jar:/Users/david/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/scodec/scodec-bits_2.13/1.1.12/scodec-bits_2.13-1.1.12.jar:/Users/david/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/typelevel/cats-macros_2.13/2.1.0/cats-macros_2.13-2.1.0.jar:/Users/david/Library/Caches/Coursier/v1/https/repo1.maven.org/maven2/org/typelevel/cats-kernel_2.13/2.1.0/cats-kernel_2.13-2.1.0.jar
```

Do you know a flag to prevent the first two line from showing up?